presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PushAggregationThroughOuterJoin.java

```diff
@@ -168,7 +168,12 @@ public Result apply(AggregationNode aggregation, Captures captures, Context cont
                     join.getDistributionType());
```

Comment:
commit message is too long, it wraps in github, please follow https://chris.beams.io/posts/git-commit/